02LessonTwo/homework/index.html

```diff
@@ -6,5 +6,26 @@
u0Eh1IIGsbnmOB4jdxAMeiOAC4bB8o8MrL/3MSeslJ78vVgHkW50V9LNiLjcHLh2AzFSYj9WRRdPfp4EjjWZ0HKy4HCMN3zdU8Hi/dnzBBrMYpOymTLKcFF9AQ2gP/6OK74sMfS4/OtVdwrK2q1tpzL9bCRt/LKBWqKnPk1j9oKfKO3/AAwHN0HxP5U3bMuWD/Fh9zofyrvlB1WBfrTR/Bh+dV+k8JfCkLvzZv5QT+VLsD5ZoU7o7s2GeaeCRXEgYYiKWIlXjDrkkOcbu0o0N733byK2Bn6yNH+sqnzI199bHQnEqMbNGT2nhiZfBHcMP8wNYnaiymPeSL+JONwXaLnGQa3uAJ0UC5Nh2ZQADyNW1xkeKjzArLG4ta1wRxDKdx7jrRDl1vyv/vWTiEQHsADcDYWBtut3CudSs6tNHnPThR1mGwwJIiRnJNyQpOVB3kAHXfoKwf2Y1iVPZW9t+4Va6SY4vjsTY6LlS/IIutuXavTRj3yEdkAix9q+vnWsjkqo1hWNptjI9kkxq+cgNbhzPvq2vR87+sBHx99MTabhAqqlhuJBvYd1O/bMtrZE8e18L6VBvL0OuMcFbp8fJINlSFSvWkDUZdbW8f0rMhwbFL3sLlRvO7Sr6bSkYDsqNxHta1CkzhcnV5rFjcG283Pxpxc1z9hTx43TSfHz8GriklkUgKtjbc6ncb8PKottEmSOX+8T3lVb4q1VfnHG27uNPxsobDxm/sOw8BmuPRXPpXZF7nlSSp0yaJ7jfurJ6Tr+5B45x8DVrDPwFVtsjMIl5yIPW9XOc9DeARpFGNyIqjyFvyqKrW0fb8vzNVaiyh1dXV1ICPEDsmsropNeDJ/ds6eQY29xAq1trFdVGH+iCM/2ToT5XB8qw+iU+kzDUGZyO8G1aS2F1CuocZhVlRkcXVhY/qO8b6mBrqyMxOie0WhkOAnNyNYHP0lP0fjbvuOVF1CnSXZPXxgr2ZY+0jbjflfh+VaHRTbvzmMh9J4uzKN1+GcDv48j4itvfcE62Noi9CuycUdn7VU7ocV2W5CQnQ/zEeUhoroe6bbL67DsQO2nbXxHDzFx42oi6YTVo9ghkuARViRLjMOO/uoL+TrpB86wsbk3a2R+6VdCT9oWb7wo1wz8OdXOdFSSM71OVuf6jjQZ0YxnzTFYzByA5RnxkAAuera5lRR3NuHG5r0GTDg7tKBvlFgMHzfHqDnwcil7Xu2HkISUd+8eGtAwS2tiQ20oXsR1sE8NuGaMmTQ21FiutbdVenaKk2ClW2VcQY7gaZZ1OvnarVSyclcXAHdOzeXCLzkB9Cv61oTAdUAxAuba8b6WHrWX04BOKwgUgG72JBIvdd4uL+tWv2MHT947MSL2UBBcag9ntGxsdWO6s9EPqzM6LP8Auch3xu6H1v8AnWbPjHw+J+eRnMEm6phfgI108x1g+7V7Y5yYrEx8ysg89/4h6UvQXDRTYmfB4kfusWXVWG9cRESyFTwNmcd+a3GtJW2ZbpI9RGNZ0GujAG1l3HUXtvqrjMSI0LWLEblUXZm4Ko5n3bzpVKPYO0cIixokWNiQBUKuIpQo3Zlfsmw5GopNtSRm02BxkfeIc6/zKbVzvFJdDrWaDXJ5xtPYOITEqps808ckzLwLXdnRTxYBb+PiL0cPiRIMqg5uX6UYdIOkUDT4OeKTtwTAOrKysI2tmuGA4Aj71V+mewkhxrowtHOOuiYaFWb+0CnjZtbcjVtGqNvkisrxzpbowIMPMQbI7AfV599t1TQYOUBgUe7d351cwePkwYKyDrI2IIlW+/d2h/751ox7awzKLyZTa5IDam1t/GuLI8kXWnb4PSw+VNJ69+zMOMuAAY25btNK5JHOmRr79BfT1rWOPhJNpNNd19/dpp/tTcPiUDsQ4AIUct177/Gp6nv6Tr2VVP8Ao1JJn07XLgDu8qzZoC0TKRvUN59pW+KUkuEQW7IpmE/tGFzYRyWBJNvZOnoK7UqPD12zMwsm48wKXEteXDDnNH+IVDEdT9p/xGpT/wDJwn/ej/GtdBBnpO0Pb8h+dVas7Q9vyFVqiUOrq6upAVtoxB4yp1B0PgdD8aFeii9WZoTvR7+IOl/cPWjCf2T4UJwi20Ht9KIE95uv6VpcCYUYF79jj9H9P0/3qesyteb6J4lVJ8SNaQyOhfbWHfDzDFwe0ntrwePjfy0PkeFFFV8aNBQnQM09nY5J4kljPZcX71PFT3g1OwuLUG9A2y4nFxLpGBmC8AwYC45aG3pyozptUNO0DvQOT5rtGbBnRZx1kPIOtzYcrrnH3Fr1/Dy5lB48fGvEumDFMXgpE7LiVbMN+jJXtkYs7gbrg+ZGtWi7RCSpl2PEc/WmbX2euIglhb2ZUdCeWZSLjwveo6nwp3imI8a2qXl2Jc3E2GyBgTcrJh5MjXPPKCfOtmGYOquNzqrjwYAj3GleFc22I7djrpzl4XeBS3qdayuiLk4KAnXsW8gxA9wFYycG8XLRidM9MXgz3uPetb0egHdasXp2P3mDPHrbeRKVr4c9kVN8Ip1YL7STqtoRng6tH5gm3+io8DgeskxUQYo6vHNE43o+puPUVP06FnhYbw8dj4qpPvAp+D02ibcYNf5h+grXQyendDemC4gCDEWixiWDIbAS8ni4MDvsN2vCi2vItp4COWGTrFDGNSyHUFW39lhqNw0qX5J+kGJmjkWWZpBHlC5rEgd7bz53qkZWTkqPR9q7Iw+KUrPCko1HaAuPst7Smx3givNul3R8wr81kLTQuszYGQk9bh540L9UzfSjYCw8ByvRrtuVvmWMOYgiGVgQSCCEaxBGo3Chzbc7NgNjuxJZsTggWOpIaN1a543G+tGVwefbIx140Laq1lbzOUmsbC4gRrJEwuVLgaeQ996n2fpAe57DwzLTXgUz4m4vZ9PMteoTSp2dWG9SoXC45VAAF777gHXlY7q2jiI3AyoPHT0tw4UHYlBemw1KWCMlaOiHiXB01Z//2Q==" />
```

Comment:
Ah! So in this case, your HTML would be looking for a folder called "users" inside its current folder, which is the homework folder! Step out of the current folder using "../", so it would be img src="../../archerlogo.jpg"